package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "airtable",
-  "version": "0.12.2",
```

Comment:
Do you mind revert this change and rebuild `airtable.browser.js)`? since our release process will automatically bump this version number up :)